paket.template

```diff
@@ -0,0 +1,24 @@
+type file
```

Comment:
Given that you are auto-generating an Assembly Info file, you should be able to change this to ``type project`` and delete everything down to the language from this file (as long as it's in the same directory as the related proj file).

Check the link I posted above for the assembly attributes you might want to add to make sure you have all the relevant fields.